packages/gatsby-source-contentful/src/extend-node-type.js

```diff
@@ -631,11 +690,44 @@ exports.extendNodeType = ({ type, store, cache, getNodesByType }) => {
   const sizesNode = fluidNodeType({ name: `ContentfulSizes`, getTracedSVG })
   sizesNode.deprecationReason = `Sizes was deprecated in Gatsby v2. It's been renamed to "fluid" https://example.com/write-docs-and-fix-this-example-link`
 
+  // gatsby-pugin-image
+  const getGatsbyImageData = () => {
+    // @todo check if gatsby-plugin-image is enabled
+    const gatsbyImageEnabled = false
+    if (!gatsbyImageEnabled) {
+      return {
+        type: GraphQLString,
+        deprecated: true,
+        deprecationReason: `Enable gatsby-plugin-image to use this field.`,
+        resolve: () => null,
+      }
```

Comment:
Just an idea: We could tell the user that there are more features by adding a deprecated field.

Reply:
Great idea